src/phoenix/server/api/context.py

```diff
@@ -146,3 +148,7 @@ async def log_out(self, user_id: int) -> None:
         response = self.get_response()
         response.delete_cookie(PHOENIX_REFRESH_TOKEN_COOKIE_NAME)
         response.delete_cookie(PHOENIX_ACCESS_TOKEN_COOKIE_NAME)
+
+    @cached_property
```

Comment:
Do we want caching to be this aggressive? What if a user logs out of one account and into another?

Reply:
i think this a per-request object derived from token parsing, and should be immutable at location